odl/test/tomo/geometry/geometry_test.py

```diff
@@ -700,15 +700,51 @@ def test_cone_beam_geometry_helper():
 
     # Validate angles
     assert geometry.motion_partition.is_uniform
-    assert pytest.approx(geometry.motion_partition.extent, 2 * np.pi)
+    assert geometry.motion_partition.extent == pytest.approx(2 * np.pi)
     assert (geometry.motion_partition.cell_sides <=
             (r + rho) / r * np.pi / (rho * omega))
 
     # Validate detector
     det_width = 2 * magnification * rho
     R = np.hypot(r, det_width / 2)
     assert geometry.det_partition.cell_sides <= np.pi * R / (r * omega)
-    assert pytest.approx(geometry.det_partition.extent, det_width)
+    assert geometry.det_partition.extent == pytest.approx(det_width)
+
+
+def test_helical_geometry_helper():
+    """Test that helical_geometry satisfies the sampling conditions.
+
+    See the `helical_geometry` documentation for the exact conditions.
+    """
+    # Parameters
+    src_radius = 3
+    det_radius = 9
+    num_turns = 4
+
+    magnification = (src_radius + det_radius) / src_radius
+    rho = np.sqrt(2)
+    omega = np.pi * 10.0
+    r = src_radius + det_radius
+
+    # Create object
+    space = odl.uniform_discr([-1, -1, -2], [1, 1, 2], [20, 20, 40])
+    geometry = odl.tomo.helical_geometry(space, src_radius, det_radius,
+                                         num_turns=num_turns)
+
+    # Validate angles
+    assert geometry.motion_partition.is_uniform
+    assert (geometry.motion_partition.extent ==
+            pytest.approx(num_turns * 2 * np.pi))
```

Comment:
Same comment: some duplication with `test_cone_beam_geometry_helper` (which is perhaps ok for testing).

Reply:
There is quite some duplication but it seems there is also quite some non-duplication :/ separating them in a natural way might be hard so I guess I'll just leave it for now

Reply:
Agreed, also with @olivierverdier's comment in parens. Code duplication in tests is more forgivable as long as it's not easy to factor it out into a fixture. The one thing we certainly don't want is a bug in a testing helper 😉